src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -248,6 +263,60 @@ static tusb_speed_t get_speed(USB_OTG_DeviceTypeDef* dev)
   return (enum_spd == DCD_HIGH_SPEED) ? TUSB_SPEED_HIGH : TUSB_SPEED_FULL;
 }
 
+
+# if defined(EMBEDDED_HS_PHY)
```

Comment:
tinyusb only accept macro `CFG_TUD_XYZ` as configure macro, and it must be platform-independent. Currently there is no direct low level device driver from applictaion. `EMBEDDED_HS_PHY` should be detected based on the actual device name i.e  `STM32F723xx` somewhere near the EP_MAX_HS/FS macro definition.

Reply:
Can I use # if defined(USB_HS_PHYC)?
USB_HS_PHYC is defined in stm32f723xx.h and so only available for the STM32F723 for now. But it would be ready for coming new devices with the same PHYC

Reply:
Yeah absolutely, using the USB_HS_PHYC is definitely an correct way to do so, it is only defined on MCU with built-in HS PHY. Bravos !!!